README.md

```diff
@@ -67,7 +70,43 @@ pip install [--user] lyncs_io[mpi]
 
 ## Documentation
 
-The high-level `load` and `save` (or `dump` as alias of `save`) functions provided by the Lyncs IO can be used as follows:
+The package provides three high-level functions:
+- `head`: loads the metadata of a file (e.g. `shape`, `dtype`, etc)
+- `load`: loads the content of a file
+- `save` or `dump`: saves data to file
+
+```python
+import numpy as np
+import lyncs_io as io
+
+arr1 = np.random.rand(10,10,10)
+io.save(arr1, "data.npy")
+
+arr2 = io.load("data.npy")
+
+assert (arr1 == arr2).all()
+```
+
+NOTE: for `save` we use the order `data, filename`. This is the opposite
+of what done in `numpy` but consistent with `pickle`'s `dump`. This order
+is preferred because the function can be used directly as a method
+for a class since `self`, i.e. the `data`, would be passed as the first
+argument of `save`.
+
+### Supported file formats
+
+Format | Extensions | Binary | Archive | Parallel MPI | Parallel Dask
+-------|------------|--------|---------|--------------|--------------
+pickle | pkl        | yes    | no      | no           | no
+dill   | dll        | yes    | no      | no           | no
+JSON   | json       | no     | no      | no           | no
+ASCII  | txt        | no     | no      | no           | no
+Numpy  | npy        | yes    | no      | yes          | yes
+Numpyz | npy        | yes    | yes     | TODO         | TODO
+HDF5   | hdf5,h5    | yes    | yes     | yes          | yes
```

Comment:
HDF5 with Parallel Dask is still TODO right?

Reply:
Good catch, thanks!